packages/graphql/tests/integration/issues/557.int.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { gql } from "apollo-server";
+import neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src/classes";
+
+describe("https://github.com/neo4j/graphql/issues/557 - Input Object type ArticleCreateInput must define one or more fields", () => {
```

Comment:
:thinking: I'm not sure why using a separate file (`557`) instead of adding these tests to the (`556`) as we are testing the same piece of work.

I'm happy to leave it as is, just pointing out that seems a bit weird

Reply:
556 doesn't exist yet (it's a schema test), but you're absolutely right, renaming to 556 now.